Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -992,19 +992,20 @@ WI.NetworkManager = class NetworkManager extends WI.Object
                 return;
 
             case WI.LocalResourceOverride.InterceptType.Request: {
+                let method = localResource.requestMethod ?? (isPassthrough ? request.method : "");
```

Comment:
Why the empty string?

Reply:
if the developer does not provide a HTTP method and has not enabled passthrough then we will interpret that as "remove the HTTP method" (just like how if no HTTP headers are provided and passthrough is not enabled then that's like saying "remove all headers")

the reason we have the passthrough option is to allow developers to differentiate between "please remove this value" and "please use whatever this value would be if the local override didn't exist"

Reply:
I guess I was wondering why this wasn't null or some such. But if empty string has that meaning, so be it.